build/BUILD.ada-url

```diff
@@ -0,0 +1,13 @@
+
+cc_library(
+    name = "ada-url",
+    hdrs = ["ada.h", "ada_c.h"],
+    srcs = ["ada.cpp"],
+    visibility = ["//visibility:public"],
+    include_prefix = ".",
+    copts = ["-w", "-msse2"],
```

Comment:
Unfortunately, this will need special handling: `-msse2` should not be passed on ARM builds, so let's use a select() statement based on architecture here. However, x86_64 always includes SSE2 (the 64-bit extension was introduced some time after SSE2), so the flag should not be needed, a select statement for ADA_SSE2 might be enough.

Reply:
Yeah, that was just experimental trying to see what would work.

Reply:
ADA_SSE2 should still be removed here – otherwise this may break the arm64 build.

Reply:
arm64 build appears to work just fine with this thankfully.